src/frontend/download-chrome.ts

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env node
+import * as os from "os";
+import { promisify } from "util";
+import * as fs from "fs";
+import got from "got";
+import stream from "node:stream";
+import extract from "extract-zip";
+import { unreachable } from "./src/utils/utils";
+
+const pipeline = promisify(stream.pipeline);
+
+const stat = promisify(fs.stat);
```

Comment:
which version of NodeJs is used? you cannot import the promises "directly"?

```
import {stat} from 'fs/promises';
```

btw. isn't `stat` deprecated?

```
import {access} from 'fs/promises';

try {
   await access(moduleExecutablePath)
   ...
```

Reply:
thanks, updated to fs/promises! imports were copied over from original script. will keep `stat` for now, just because it works